drivers/hmis/app/models/hmis/ce/referral_message_handler.rb

```diff
@@ -75,13 +77,22 @@ def accept_referral
     end
 
     def reject_referral
+      disable_outstanding_steps(referral)
       referral.completed_at = Time.current
       referral.reject!
       set_custom_referral_status(status_key: 'rejected')
       referral.opportunity.release!
       mark_client_dirty(referral.client)
     end
 
+    def disable_outstanding_steps(referral)
```

Comment:
other methods use `self.referral` rather than passing it as an arg. Suggest removing this to keep the style pattern consistent
```suggestion
    def disable_outstanding_steps
```